### PR TITLE
Implement the auto comment parameter to the commands. Fix fetch-credential bug

### PR DESCRIPTION
**Pull request by suhasini-b**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DelineaSS/ReleaseNotes/3_0_0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+##### DelineaSS
+- Update the implementation of the fetch-credentials.
+- Updated the pack with autoComment feature for commands.
+  - delinea-secret-rpc-changepassword
+  - delinea-secret-delete
+  - delinea-secret-password-update
+  - delinea-secret-get
+
+- Updated the Docker image to: *demisto/python3:3.10.11.59070*.
```
Comment (merit-maita) [inline Packs/DelineaSS/ReleaseNotes/3_0_0.md:12]:
```suggestion
- Update implementation of the ***fetch-credentials*** command.
- Added the *autoComment* argument to the following commands.
  - ***delinea-secret-rpc-changepassword***
  - ***delinea-secret-delete***
  - ***delinea-secret-password-update***
  - ***delinea-secret-get***

- Updated the Docker image to: *demisto/python3:3.10.11.59070*.
```

**Review by merit-maita — CHANGES_REQUESTED** [review]
looks very good, thanks for the effort!
please just address my comment regarding the RN.

File: Packs/DelineaSS/ReleaseNotes/3_0_0.md (hunk 1 above)
Comment (suhasini-b) [reply Packs/DelineaSS/ReleaseNotes/3_0_0.md]: @merit-maita I have addressed the comment. Please take a look

**Comment by suhasini-b** [comment]
Test_cred_on_instance2
fetch_cred_instance1.

On instance 1 we have catch the credential 5138 secret ID and test the credential on instance 2. Please find the attach screenshot.

**Review by merit-maita — APPROVED** [review]

**Pull request merged**